docker/scout/scripts/main.py

```diff
@@ -751,22 +751,22 @@ def update_bpt_gauge(bpt_gauge: Gauge, amm_gauge: Gauge, bpt_name: str, bpt_addr
             AMM_BALANCER,
             "price"
         ).set(usd_prices_by_token_address[token_address])
-        token_balance = balances[index] / (
+        usd_token_balance = balances[index] / (
                 10 ** bpt_underlying_token.decimals()) * usd_prices_by_token_address[token_address]
         bpt_gauge.labels(
             bpt_name,
-            bpt_name,
-            bpt_address,
+            token_symbol,
+            token_address_checksummed,
             "usd_balance"
-        ).set(token_balance)
```

Comment:
this looks like a USD balance global to the pool and not on tokens, bnut it seems like we are setting one token balance and usd balance?